pkg/jx/cmd/get_pipeline.go

```diff
@@ -65,36 +70,91 @@ func NewCmdGetPipeline(f Factory, in terminal.FileReader, out terminal.FileWrite
 
 // Run implements this command
 func (o *GetPipelineOptions) Run() error {
-	jenkins, err := o.JenkinsClient()
+
+	_, _, err := o.JXClient()
```

Comment:
What is the reason for creating an empty JxClient?

Reply:
Answered below.